controlsfx/src/main/java/org/controlsfx/control/table/TableFilter.java

```diff
@@ -78,7 +78,7 @@ private TableFilter(TableView<T> tableView, boolean isLazy) {
         tableView.setItems(sortedControlList);
 
         applyForAllColumns();
-        tableView.getStylesheets().add("/impl/org/controlsfx/table/tablefilter.css");
+        tableView.getStylesheets().add(FilterPanel.class.getResource("/impl/org/controlsfx/table/tablefilter.css").toExternalForm());

```

Comment:
I am nitpicking here :)

Can you change `FilterPanel.class` to `TableFilter.class` ?

Reply:
Of course. It was my mistake....